src/components/Sidebar/Sidebar.sass

```diff
@@ -5,17 +5,20 @@
   height: auto
   width: 100%
   overflow: scroll
-  position: relative;
+  position: relative
+
+  li
+    font-size: 1.5rem
 
   p
-    font-size: 1.2rem
+    font-size: 1.5rem
     font-style: italic
     color: $dark-gray
     margin-bottom: 1rem
     +bold
-
   .fixed-search
```

Comment:
Curious why you need tailored styles for `.fixed-search` here... It seems like something we should be able to avoid since it's coupled to `Sidebar`, which should be the same everywhere it's used, just with variable content.

Let's talk about this one..